src-backbone/app/js/behaviors/authentication_form.js

```diff
@@ -0,0 +1,49 @@
+var Helpers = require('utils/helpers');
+
+module.exports = Marionette.Behavior.extend({
+
+    ui: {
+        form: 'form'
+    },
+
+    events: {
+        'submit @ui.form': 'auth',
+    },
+
+    show_errors_in_form: function($form, errors) {
+        $form.find('.alert').remove();
+        $form.find('.has-error').removeClass('has-error');
+        $form.find('span.help-block').remove();
+
+        Object.keys(errors).forEach(function(key) {
+            if (key === '__all__') {
```

Comment:
That's a default key that Django forms return when there's a validation error for the entire form. e.g. Invalid username/password